tests/test_operator.py

```diff
@@ -950,6 +953,37 @@ def test_incomplete_override(self):
 
 class TestDeclarator(object):
 
+    def test_conditional_declarations(self):
+        accesses = [[0, 0], [0, 1], [1, 0], [1, 1]]
+        dims = len(accesses[0])
+        pts = len(accesses)
+        stencil_name = 's%dd_%s_%dpt' % (dims, 'name', pts)
```

Comment:
Do we really need this complicated name? can we not just use `'a'` ?

Reply:
Makes sense. Changed.